chain-information.json.example

```diff
@@ -0,0 +1,34 @@
+{
+  "chainName": "<CHAIN NAME>",
+  "description": "<SHORT ONE LINE DESCRIPTION OF THE CHAIN>",
+  "chainId": ######,
+  "platformChainId": "<P CHAIN ID>",
+  "subnetId": "<SUBNET ID>",
+  "vmId": "<VM ID>",
+  "vmName": "EVM",
+  "subnetExplorerUriId": "<SHORT-CHAIN-NAME>",
+  "explorerUrl": "https://subnets.avax.network/<SHORT-CHAIN-NAME>",
+  "rpcUrl": "https://subnets.avax.network/<SHORT-URL-TAG>/testnet/rpc",
+  "wsUrl": "wss://subnets.avax.network/<SHORT-URL-TAG>/testnet/ws",
+  "isTestnet": true,
+  "networkToken": {
+    "name": "<TOKEN-NAME>",
+    "symbol": "<TOKEN-TICKER>",
+    "description": "<NETWORK NATIVE TOKEN DESCRIPTION>",
+    "decimals": 18 OR 9,
```

Comment:
Aren't clients able to choose any decimal setting for there native token?  Or is this actually limited to 18 or 9?

Reply:
They might be able to but I'm not sure if or explorer supports it. 
It's a pretty advanced configuration change that a team should really know what they are doing if they don't use these defaults since there may be unintended consequences. If they integrate with CoinGecko for example, this might become an issue.
Since it's an example and not our API docs I would prefer to be more prescriptive, to help people fill it in correctly when they don't really care, then to accurately represent the full range of possibilities.

Reply:
If this is for the native coin of the network, this should be 18 and we should not allow people to modify it.

If this is intended for arbitrary ERC-20s, then there are no real restrictions other than the requirements from the ERC-20 spec: https://eips.ethereum.org/EIPS/eip-20.

Reply:
This would be for the native token.